cocotb/share/lib/utils/cocotb_utils.c

```diff
@@ -66,7 +66,19 @@ void* utils_dyn_open(const char* lib_name)
     SetErrorMode(0);
     ret = LoadLibrary(lib_name);
     if (!ret) {
-        printf("Unable to open lib %s\n", lib_name);
+        LPTSTR msg_ptr;
+        FormatMessage
+        (
+            FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_ALLOCATE_BUFFER,
+            NULL,
+            GetLastError(),
+            MAKELANGID(LANG_NEUTRAL, SUBLANG_SYS_DEFAULT),
+            (LPTSTR)&msg_ptr,
+            255,
+            NULL
+        );
```

Comment:
Please format this function call more like our existing function calls, so 

```
FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM | 
              FORMAT_MESSAGE_ALLOCATE_BUFFER, NULL, 
              GetLastError(), ...);
```

Reply:
I condensed the formatting and added check of return value. Also provided more detail in git commit message.